controllers/tasks.js

```diff
@@ -130,11 +132,31 @@ const fetchPaginatedTasks = async (query) => {
 
 const fetchTasks = async (req, res) => {
   try {
-    const { dev, status, page, size, prev, next, q: queryString, assignee, title } = req.query;
+    const {
+      dev,
+      status,
+      page,
+      size,
+      prev,
+      next,
+      q: queryString,
+      assignee,
+      title,
+      "assignee-role": assigneeRole,
+      "next-archived-user": nextArchivedUser,
```

Comment:
- What's next-archived-user?
- Why do we need to pass user id to fetch tasks?

Reply:
please refer [this](https://github.com/Real-Dev-Squad/website-backend/pull/1748#discussion_r1422366360)